SampleApp/SearchPinsViewController.swift

```diff
@@ -9,16 +9,23 @@
 import Pelias
 import TangramMap
 
-class SearchPinsViewController: SampleMapViewController, UITextFieldDelegate {
+class SearchPinsViewController: SampleMapViewController, UITextFieldDelegate, AutocompleteSearchDelegate {
 
   @IBOutlet weak var searchField: UITextField!
 
+  @IBOutlet weak var displaySearch: UIButton!
+  let searchListSegueId = "searchListSegueId"
+
   override func viewDidLoad() {
     super.viewDidLoad()
 
-    try? loadStyle(.bubbleWrap)
+    try? loadStyleAsync(.bubbleWrap) { [unowned self] (style) in
```

Comment:
The degree of "incorrectness" here depends entirely on your application :)

A synchronous scene load is blocked on I/O operations (potentially network calls) and you usually don't want to do that on a UI thread. We mostly offer this synchronous method so that you can make it asynchronous using a different mechanism if you want to.